src/NBitcoin/Block.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using NBitcoin.BouncyCastle.Math;
+﻿using NBitcoin.BouncyCastle.Math;
```

Comment:
So we agreed Systems namespaces first, then the other namespaces.

Reply:
Ah right I need to configure the vs tools to do that